samples/bluetooth/scanning_while_connecting/central/README.rst

```diff
@@ -0,0 +1,238 @@
+.. _bt_scanning_while_connecting:
```

Comment:
This is probably buried too deep in the structure. Suggesting to leave out the `central` folder.

Reply:
Thank you for your feedback @peknis. The doc build fails because there are references to another sample which is also still in review. Apart from that, I think you can look at the documentation again.